src/common/components/ReviewModal/ReviewModal.tsx

```diff
@@ -71,11 +76,12 @@ export const ReviewModal = ({
           {/* seeMore link only shown when user is from default reviews page, hidden when in professor/course pages */}
           {seeMore && (
             <div className={seeMoreWrapper()}>
-              <div className={seeMoreLink()}>
-                {/* TODO: link to respective prof/course page */}
-                {/* can use review.professorName for professors and review.courseCode for courses */}
+              <Link
+                href={`${reviewPath.replace(/\s+/g, "-")}`}
```

Comment:
issue: professor links are not guaranteed to be their name with the space replaced to dash

suggestion: we'll need more changes to include professor's `slug` (which is used in professor links like `/professor/:slug`) in the review object

specifically in these areas:

- updating the review type -
    - [`src/common/types/review.ts`](https://github.com/AfterClass-io/afterclass.io-v2/blob/main/src/common/types/review.ts)
- updating the return object from review apis -
    - [`src/server/api/routers/reviews.ts#L153-L170`](https://github.com/AfterClass-io/afterclass.io-v2/blob/main/src/server/api/routers/reviews.ts#L153-L170)
    - [`src/server/api/routers/reviews.ts#L196-L215`](https://github.com/AfterClass-io/afterclass.io-v2/blob/main/src/server/api/routers/reviews.ts#L196-L215)